src/app/pages/rpc-send-transfer/use-rpc-send-transfer.ts

```diff
@@ -32,13 +33,17 @@ export function useRpcSendTransfer() {
   if (!origin) throw new Error('Invalid params');
 
   return {
-    address,
-    amount,
+    recipients: recipients.map((address, index) => ({
+      address,
+      amount: amounts[index],
+    })),
     origin,
     utxos,
+    totalAmount: sumNumbers(amounts.map(Number)),
```

Comment:
What does passing number as the callback do 🤔

Reply:
converts to num